lib/internal/util.js

```diff
@@ -4,6 +4,13 @@ const binding = process.binding('util');
 const prefix = `(${process.release.name}:${process.pid}) `;
 const noDeprecation = process.noDeprecation;
 
+// This constructor is used for creating objects for general storage purposes.
+// Instantiating this is faster than explicitly calling `Object.create(null)`
+// to get a "clean" empty object (tested with v8 v4.9).
+function StorageObject() {}
+StorageObject.prototype = Object.create(null);
```

Comment:
I'd strongly recommend defining `StorageObject.prototype.toString`, so that stringification of these objects doesn't throw.